tests/raftstore/test_stale_peer.rs

```diff
@@ -57,14 +57,17 @@ fn test_stale_peer_out_of_region<T: Simulator>(cluster: &mut Cluster<T>) {
     // Isolate peer 2 from other part of the cluster.
     cluster.add_send_filter(IsolationFilterFactory::new(2));
```

Comment:
No matter 2 is leader or not, L62 can ensure that following operations can be executed properly.